HW7/OlenaBudko/Task_07_1.py

```
@@ -0,0 +1,49 @@
+    """ Calculate the largest number """
+    return a if a > b else b
+
+
+print(max_number(4, 7))
+
+
+# Task 2
+def rectangle():
+    a = float(input("Enter first side by rectangle: "))
+    b = float(input("Enter second side by rectangle: "))
+    print(f"The area of rectangle is: ", a * b)
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)